ang/afsearchAipErrorLogs.aff.json

```diff
@@ -0,0 +1,25 @@
+{
+    "type": "search",
+    "requires": [],
+    "entity_type": null,
+    "join_entity": null,
+    "title": "Aip Error Logs",
+    "description": "",
+    "is_dashlet": true,
+    "is_public": false,
+    "is_token": false,
+    "contact_summary": null,
+    "summary_contact_type": null,
+    "summary_weight": null,
+    "icon": "fa-list-alt",
+    "server_route": "civicrm/aip_error_logs",
+    "permission": [
+        "access CiviCRM"
```

Comment:
Should this be `administer CiviCRM` in correspondence to the permissions in https://github.com/systopia/aip/blob/c935e72cd2c966e6574ac22f9f9f522df9db1e5d/Civi/Api4/AipErrorLog.php#L13 ?

Reply:
I changed this